pkg/sdkserver/sdk.go

```diff
@@ -77,5 +77,16 @@ func convert(gs *agonesv1.GameServer) *sdk.GameServer {
 		}
 	}
 
+	if runtime.FeatureEnabled(runtime.FeatureCountsAndLists) {
+		if gs.Status.Counters != nil {
+			counters := make(map[string]*sdk.GameServer_Status_CounterStatus, len(gs.Status.Counters))
+			for key, counter := range gs.Status.Counters {
+				tmpCounter := counter.DeepCopy()
+				counters[key] = &sdk.GameServer_Status_CounterStatus{Count: tmpCounter.Count, Capacity: tmpCounter.Capacity}
```

Comment:
rather than counter.DeepCopy(), I'd be tempted to replace line 85 with:

```
counters[key] = &sdk.GameServer_Status_CounterStatus{Count: proto.Int32(counter.GetCount()), Capacity: proto.Int32(counter.Capacity())}
```

it's more idiomatic for proto manipulation.

Reply:
Counter doesn't have `counter.GetCount()` or `counter.Capacity()` Are those methods I should implement?